packages/integration-sdk-cli/src/commands/types.ts

```diff
@@ -0,0 +1,154 @@
+import * as log from '../log';
+import { loadConfig } from '../config';
+import * as path from 'path';
+import { buildStepDependencyGraph } from '@jupiterone/integration-sdk-runtime';
+import { createCommand } from 'commander';
+import {
+  IntegrationStepExecutionContext,
+  Step,
+  StepGraphObjectMetadataProperties,
+  StepEntityMetadata,
+  StepRelationshipMetadata,
+} from '@jupiterone/integration-sdk-core';
+import { promises as fs } from 'fs';
+
+export interface TypesCommandArgs {
+  projectPath: string;
+  typesFilePath: string;
+}
+
+export function types() {
+  return createCommand('types')
+    .description('Collects types metadata for all steps')
+    .option(
+      '-p, --project-path <directory>',
+      'Absolute file path to the integration project directory. Defaults to the current working directory.',
+      process.cwd(),
+    )
+    .option(
+      '-t, --types-file-path <path>',
+      'Absolute file path to the JSON file that should be created/overwritten. Defaults to {CWD}/docs/types.json.',    
+    )
+    .action(executeTypesAction);
+}
+
+export async function executeTypesAction(
+  options: TypesCommandArgs,
+): Promise<void> {
+  const { projectPath } = options;
+  const typesFilePath =
+    options.typesFilePath ||
+    getDefaultTypesFilePath(projectPath);
+
+  log.info('\nFetching metadata types from steps...\n');
```

Comment:
How about `Collecting metadata types from steps...\n'` since the command is described as a collector?